docs/src/modules/components/AdDisplay.js

```diff
@@ -1,15 +1,24 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import { styled } from '@mui/material/styles';
-import { adShape } from 'docs/src/modules/components/AdManager';
 import { GA_ADS_DISPLAY_RATIO } from 'docs/src/modules/constants';
 import { adStylesObject } from 'docs/src/modules/components/ad.styles';
 
-const Root = styled('span', { shouldForwardProp: (prop) => prop !== 'shape' })(({
-  theme,
-  shape,
-}) => {
-  const styles = adStylesObject[`body-${shape}`](theme);
+const InlineShape = styled('span')(({ theme }) => {
+  const styles = adStylesObject['body-inline'](theme);
+
+  return {
+    ...styles.root,
+    '& img': styles.img,
```

Comment:
Was this file manually changed?

Reply:
Yes, this one is required to be changed manually.